airflow/example_dags/tutorial_taskflow_api_pandas.py

```diff
@@ -0,0 +1,153 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=missing-function-docstring
+"""
+### TaskFlow API Example using Pandas
+
+This is a simple ETL data pipeline example which demonstrates the use of the
+TaskFlow API using three simple tasks for Extract, Transform, and Load, while
+using Pandas and loading data from files.
+
+"""
+# [START tutorial]
+# [START import_module]
+import pandas as pd
+
+# The DAG object; we'll need this to instantiate a DAG
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.sensors.filesystem import FileSensor
+from airflow.utils.dates import days_ago
+
+# [END import_module]
+
+
+def generate_data_file():
+    tmp_data_file = open('/tmp/order_data.csv', 'w')
+    tmp_data_file.write('order_id,order_value\n')
+    tmp_data_file.write('"1001", 301.27\n')
+    tmp_data_file.write('"1002", 433.21\n')
+    tmp_data_file.write('"1003", 502.22\n')
+    tmp_data_file.close()
+
+
+# [START default_args]
+# These args will get passed on to each operator
+# You can override them on a per-task basis during operator initialization
+default_args = {
+    'owner': 'airflow',
+}
+# [END default_args]
+
+
+# [START instantiate_dag]
+with DAG(
+    'tutorial_taskflow_api_pandas',
+    default_args=default_args,
+    description='TaskFlow API ETL Pandas',
+    schedule_interval=None,
+    start_date=days_ago(2),
+    tags=['example'],
+) as dag:
+    # [END instantiate_dag]
+
+    # [START documentation]
+    dag.doc_md = __doc__
```

Comment:
This isn't needed any more I think - the decorator got changed to do this automatically I think.

Right @casassg ?

Reply:
We would need to use the `@dag` decorator for that. I would strongly suggest doing so instead as its more in the TaskFlow API